source/includes/mongodb-compatibility-table-php.rst

```diff
@@ -13,6 +13,16 @@
      - MongoDB 4.0
      - MongoDB 3.6
 
+   * - ext + lib 1.21
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
```

Comment:
This seems identical to "ext + lib 1.20". Should that instead be changed to "ext + lib 1.20 to 1.21"?

Reply:
good call!

Reply:
will make the according changes for both tables.